React/src/types/model/Employee.ts

```diff
@@ -6,7 +6,7 @@ export type Employee = {
     password: string
     username: string
     status: string
-    roles: string[]
+    roles: string[] | string | null
```

Comment:
Zróbmy już to albo tablicę albo samego stringa, bo w sumie nie widzę zastosowania dla stringa i nulla jeżeli jest tablica która może być pusta albo zawierać jednego stringa. Teraz w końcu user ma 1 rolę czy wiele ?